components/Skeleton/Skeleton.tsx

```diff
@@ -30,53 +38,22 @@ export const Skeleton = styled('div', {
 
   variants: {
     variant: {
-      avatar1: {
-        borderRadius: '$round',
-        height: '$3',
-        width: '$3',
-      },
-      avatar2: {
-        borderRadius: '$round',
-        height: '$5',
-        width: '$5',
-      },
-      avatar3: {
-        borderRadius: '$round',
-        height: '$6',
-        width: '$6',
-      },
-      avatar4: {
-        borderRadius: '$round',
-        height: '$7',
-        width: '$7',
-      },
-      avatar5: {
-        borderRadius: '$round',
-        height: '$8',
-        width: '$8',
+      square: {
+        borderRadius: '$2',
       },
-      avatar6: {
-        borderRadius: '$round',
-        height: '$9',
-        width: '$9',
-      },
-      text: {
-        height: '$1',
-      },
-      title: {
-        height: '$5',
-      },
-      heading: {
-        height: '$3',
+      circle: {
+        borderRadius: '$round'
       },
       badge: {
         borderRadius: '$pill',
-        height: '$4',
       },
       button: {
-        borderRadius: '$1',
-        height: '32px',
-        width: '128px',
+        borderRadius: '$3',
+      },
+      text: {
+        '&:empty:before': {
```

Comment:
`empty` seems safe enough for our usage context, but might be clunky in some scenarii:

![image](https://user-images.githubusercontent.com/3367393/146351162-807b4645-fe81-4668-8496-537c05a20380.png)


https://developer.mozilla.org/en-US/docs/Web/CSS/:empty

@seedy please add a `FIXME` comment just in case

Reply:
Here's a sandbox showing that we should be fine with JSX: https://codesandbox.io/s/kind-ives-4hpeg